### PR TITLE
Experience/11133/homepage

### LINKED ISSUE #11133: Website Redesign - Homepage V2
As a User,
I want to visit the ReportStream Homepage,
So that I can become informed about ReportStream

Designs
- Figma Link https://www.figma.com/proto/Du7feM8huRDAAZCQcUIj93/Website-Redesign?node-id=1732-4121&starting-point-node-id=1732%3A4121&t=6emmL3wLwb3ofQLf-1
Content Updates
-  [See Figma](https://www.figma.com/proto/Du7feM8huRDAAZCQcUIj93/Website-Redesign?node-id=1732-4121&starting-point-node-id=1732%3A4121&t=6emmL3wLwb3ofQLf-1)
Page Title
- ReportStream
Analytics
- All existing tracking to stay
Storybook updates
- TBD
Verification Steps
    - Code Review
    - Trail / Dev environment signoff by Product / Design
    - Staging environment signoff by Product / Design / Stakeholders (Need to be explicit)
    - Production environment signoff by Engineering, Product and Design

Acceptance Criteria:
- [ ] Analytics / Metrics Required 
- [ ] Storybook update required
- [ ] Unit Tests
- [ ] End to End Tests
- [ ] Manual Testing Steps

Dependencies:
- Designs - https://www.figma.com/proto/Du7feM8huRDAAZCQcUIj93/Website-Redesign?node-id=1732-4121&starting-point-node-id=1732%3A4121&t=6emmL3wLwb3ofQLf-1
- Platform API Endpoints - N/A

Additional Details:
- Previous homepage ticket https://app.zenhub.com/workspaces/web-experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/10711
- The deployment strategy will follow the general deployment strategy for the Website Redesign EPIC. We will deliver the homepage in isolation from the other pages.

### PR DESCRIPTION
**Pull request by penny-lischer**
This PR ...

Updates styles per latest design review.

Test Steps:
1. Log into RS
2. View homepage

## Changes
Design changes from current version

-  Set max-width to desktop-lg (1200px / 75rem)
-  Reduce vertical spacing within each section from 72px to 64px
-  Change background colors from #D9E8F6 to #eff6fb or blue-5
-  Change text size of "Still exploring..." callout from H1 to H2
-  Change button size to usa-button-big with a spacing of 8px between icon and text for all buttons EXCEPT the See all partners button
-  Remove blue-5 background from "How we help you" section
-  Add blue-5 background to "How it works" section
-  Remove blue line divider between "How it works" and "Where we're live" sections
-  Replace "How it works" image with new diagram

New additions to current version

-  Add header background image
-  Add images to "How we help you" section and remove line rules
-  Add "Accepted data types..." section back into the "How it works" section

### Testing
- [x] Tested locally?
- [ ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [x] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [ ] Added tests?

## Linked Issues
- Fixes #11133

### CODE REVIEW

**Comment by audreykwr** [comment]
Two small things from me. One that we must missed last time and one that's just my mistake

- [ ] For "expert support team" should be reportstream.cdc.gov/support (that must have slipped through the PRs for the current version)
- [ ] This one is my b - test results needs to be remove from the first sentence under How it works. So it would read "securely delivers healthcare data from organizations"..

File: frontend-react/src/shared/Card/Card.module.scss [hunk 1]
```diff
@@ -5,24 +5,34 @@
...
-        @include u-padding-left(0);
-    }
-    :global(.usa-card__body) {
-        @include u-padding-left(0);
-    }
-    :global(.usa-card__footer) {
-        @include u-padding-left(0);
-    }
+    :global {
+        .usa-card__header {
```
Comment (etanb) [inline frontend-react/src/shared/Card/Card.module.scss:9]: Small thing here: can we use the SASS `&` for this BEM style CSS?

ie:

```
.usa-card {
  &__header {
    // styles
  }
  
  &__body {
    // styles
    &:last-child {
      // styles
    }
  }
}
```

File: frontend-react/src/shared/Card/Card.module.scss [hunk 2]
```diff
@@ -5,24 +5,34 @@
...
+            @include u-padding-left(0);
+        }
 
-    :global(.usa-card__container) {
-        border-style: none;
-    }
+        .usa-card__heading {
```
Comment (etanb) [inline frontend-react/src/shared/Card/Card.module.scss:13]: what is the difference between `.usa-card__header` and `.usa-card__heading`? can we make them semantically more meaningful?

Comment (penny-lischer) [reply frontend-react/src/shared/Card/Card.module.scss:13]: I didn't create them. The card component is part of uswds component library, I'm just overriding them.

**Comment by etanb** [comment]
The background image expands / contracts with the window:


https://github.com/CDCgov/prime-reportstream/assets/516877/a5cba0d2-a562-4aac-b5df-a700bbf0617f

If you apply some CSS rule like the following it should fix it:

```
background-position: right center;
background-size: contain;
background-repeat: no-repeat;
```

**Review by etanb — APPROVED** [review]

**Comment by jillian-hammer** [comment]
i know both eng reviews haven't happened yet, so this a premature check, but flagging these two things @penny-lischer 

- [ ] Connect now button should open in a new tab - I think Joseph wrote some code that would open external links in a new tab auto-magically
- [ ] The image assets (other than the header) are low quality – I exported them and shared them as a [comment on the original issue](https://app.zenhub.com/workspaces/web-experience-607d9d5e68b95200150fec37/issues/gh/cdcgov/prime-reportstream/11133#issuecomment-1698141640), but let me know if you'd like me to share them a different way

**Review by jpandersen87 — APPROVED** [review]

**Review by jillian-hammer — CHANGES_REQUESTED** [review]
thanks @penny-lischer this looks so good! two small notes:
- [x] "Connect with us" button at the bottom CTA banner: usa-button-big with a spacing of 8px between icon and text
- [ ] "Accepted data types" header too large; decrease to H3 or `22px`

**Review by jillian-hammer — APPROVED** [review]
thanks for making those changes! looks perfect @penny-lischer!

**Pull request merged**